Assign2/main.py

```diff
@@ -41,7 +43,8 @@ def experiment(injection_tubing_diameter, num_wells, num_connections, mass_flow_
     # ------------------------------------------------------------------------------------------------------------------ #
 
     ## 12 Compressor ------------------------------------------------------------------------------##
-    press_out = 345 # pipes_press_out * 6.89476  #
+    #press_out = 345 
+    press_out = pipes_press_out# * 6.89476  #
     p2 = 700 #kPa
     test_temp_out = pipes_temp_out # 300
```

Comment:
just confirming if we need to convert to kelvin here or not. If so, convert, if not its fine as is.

Reply:
Output is set to kelvin